src/pudl_archiver/cli.py

```diff
@@ -55,6 +55,11 @@ def parse_main():
         type=str,
         help="Generate a JSON archive run summary",
     )
+    parser.add_argument(
+        "--skip-publish",
```

Comment:
non-blocking: I wonder if requiring review should be the default behavior and auto-publishing should be the flagged behavior? Seems a little safer, but I'll leave that up to your judgment.